Alisha/data/lab1.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        print("'mplfinance' not found. Skipping candlestick chart. Install it using <code>pip install mplfinance</code>.")
</pre><p>We dont need the try except here since mpl finance library will always be present as it will be installed with you run pip install -r requirements.txt file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>